robot-server/robot_server/instruments/router.py

```diff
@@ -0,0 +1,112 @@
+"""Instruments routes."""
+from typing import Optional, List, Dict
+
+from fastapi import APIRouter, status, Depends, Query
+from opentrons.protocol_engine.errors import HardwareNotSupportedError
+
+from robot_server.hardware import get_hardware
+from robot_server.service.json_api import (
+    SimpleMultiBody,
+    PydanticResponse,
+    MultiBodyMeta,
+)
+
+from opentrons.types import Mount
+from opentrons.protocol_engine.resources.ot3_validation import ensure_ot3_hardware
+from opentrons.hardware_control import HardwareControlAPI
+from opentrons.hardware_control.dev_types import PipetteDict, GripperDict
+
+from .instrument_models import (
+    MountType,
+    PipetteData,
+    Pipette,
+    GripperData,
+    Gripper,
+    AttachedInstrument,
+)
+
+instruments_router = APIRouter()
+
+
+def _pipette_dict_to_pipette(pipette_dict: PipetteDict, mount: MountType) -> Pipette:
+    """Convert PipetteDict to local Pipette class."""
+    return Pipette(
+        mount=mount,
```

Comment:
Given that we're already using this function to "convert hardware PipetteDict interface to user-facing response type," I think we should map the hardware `Mount` to user-facing `MountType` here, too.

```suggestion
def _pipette_dict_to_pipette(pipette_dict: PipetteDict, mount: Mount) -> Pipette:
    """Convert PipetteDict to Pipette response model."""
    return Pipette(
        mount=MountType[mount.name],
```